javascript/frameworks/xsjs/lib/advanced_security/javascript/frameworks/xsjs/XSJSReflectedXssQuery.qll

```diff
@@ -17,13 +17,13 @@ class Configuration extends TaintTracking::Configuration {
   Configuration() { this = "XSJS Reflected XSS Query" }
 
   override predicate isSource(DataFlow::Node start) {
-    super.isSource(start) or
+    exists(DomBasedXss::Configuration domBasedXssConfiguration |
```

Comment:
For consistency, I think we could have the XSJS configurations extend the default configurations, but ensuring that the `isSink` predicate is overridden to only contain XSJS specific sinks.